### DISCUSSION TITLE
How to use custom arduino core

### DISCUSSION THREAD

**Discussion by umerm64** [espressif/arduino-esp32#9329] [Q&A]
I am using platformio for my project and I want to use a custom sdk (arduino core) for my project.
How do I specify the custom one in the `platform.json` file of platformio config?

I tried to update the `platform.json` file like this:

"framework": {
"arduino": {
      "package": "framework-arduinoespressif32",
      "script": "builder/frameworks/arduino.py",
      "url": "https://github.com/myprofile/arduino-esp32.git"
    },
but this doesnt download my custom sdk.

I need help in this regard please.

Thank you.

**Comment by Jason2866** [comment]
You can't use a git URL here. The URL has to point to a package (zip)
